domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt

```diff
@@ -41,6 +41,28 @@ class StoryProgressTestHelper @Inject constructor(
     )
   }
 
+  /**
+   * Creates a partial story progress for a particular profile.
+   *
+   * @param profileId the profile we are setting partial progress of the fraction story for
+   * @param timestampOlderThanOneWeek if the timestamp for this topic progress is more than one week ago
```

Comment:
The parameter mention in the comment doesn't match with the parameter

```suggestion
   * @param timestampOlderThanAWeek if the timestamp for this topic progress is more than one week ago
```